src/config.py

```diff
@@ -241,9 +241,9 @@ def load_data():
             "atk_len_diablo_infector": float(Config._select_val("char", "atk_len_diablo_infector")),
             "atk_len_diablo": float(Config._select_val("char", "atk_len_diablo")),
             "atk_len_cs_trashmobs": float(Config._select_val("char", "atk_len_cs_trashmobs")),
-            "kill_cs_trash": float(Config._select_val("char", "kill_cs_trash")),
-            "cs_town_visits": float(Config._select_val("char", "cs_town_visits")),
-            "runs_per_stash": float(Config._select_val("char", "runs_per_stash")),
+            "kill_cs_trash": bool(Config._select_val("char", "kill_cs_trash")),
+            "cs_town_visits": bool(Config._select_val("char", "cs_town_visits")),
+            "runs_per_stash": False if not Config._select_val("char", "runs_per_stash") else int(Config._select_val("char", "runs_per_stash")),
             "runs_per_repair": False if not Config._select_val("char", "runs_per_repair") else int(Config._select_val("char", "runs_per_repair")),
```

Comment:
i'd recommend learning what the purpose of this change was :)

Reply:
if you run CS & trash, your inventory gets filled up so fast, that you'll either miss items or get stuck often picking up an item 35 times without success. this could be resolved by clearing the stash BEFORE starting the dia run and having sufficient loot columns free. By default botty was clearing stash every 4th run, which was fine for pindle - for CS its a different story. I saw repair_per_run being added recently and thought stash_per_run could be similar.

Reply:
Yeah I understand this, but I ended up having to adjust how you implemented it. I'd look at the change in commit [0c7c67c](https://github.com/aeon0/botty/pull/460/commits/0c7c67c1416ffa3c898c0c20bb7f00d528e8897b) and figure out why I changed these lines.